package.json

```diff
@@ -43,19 +43,17 @@
     "node-fetch": "1.7.3",
     "nodemon": "^1.18.7",
     "raf": "^3.4.0",
-    "react": "16.3.0",
-    "react-apollo": "2.0.1",
-    "react-dom": "16.3.0",
     "request": "^2.88.0",
     "superagent": "^4.0.0",
     "tmp": "^0.0.33",
     "winston": "^3.0.0"
   },
   "devDependencies": {
-    "chai": "^3.4.1",
-    "mocha": "^5.1.3",
-    "nock": "^9.0.0",
-    "sinon": "^1.17.2",
+    "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
+    "mocha": "^5.2.0",
+    "nock": "^10.0.6",
+    "sinon": "^7.2.2",
```

Comment:
Upgraded the testing things. Also added `chai-as-promised` extensions for `chai` so we can do some easier promise verification.